mmdet/datasets/custom.py

```diff
@@ -178,6 +186,10 @@ def prepare_train_img(self, idx):
         img_info = self.img_infos[idx]
         # load image
         img = mmcv.imread(osp.join(self.img_prefix, img_info['filename']))
+        # corruption
+        if self.corruption is not None:
+            img = corrupt(img, severity=self.corruption_severity,
+                          corruption_name=self.corruption)
```

Comment:
These lines need to be formatted by yapf with the specified [style configurations](https://github.com/open-mmlab/mmdetection/blob/master/.style.yapf).

Reply:
Done